site/_static/images/project/lighthouse-seo.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="54" height="54" viewBox="0 0 54 54" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
FYI: This file exists to fix an Eleventy error that kept asking for lighthouse-seo.svg. I did a similar thing re: Workbox (workbox.svg vs workbox-modules.svg). I'd like a fix for it since it's not great to duplicate assets, but I just need to get things rolling.